docs/reST/ref/key.rst

```diff
@@ -269,6 +269,46 @@ for ``KMOD_NONE``, which should be compared using equals ``==``). For example:
 
    .. ## pygame.key.get_pressed ##
 
+.. function:: get_just_pressed
+
+   | :sl:`returns a pygame.key.ScancodeWrapper containing the most recent key presses`
+   | :sg:`get_just_pressed() -> bools`
+
+   Returns a sequence of boolean values representing the most recent key presses.
+   done by tracking the KEYDOWN events when the event loop is pumped using pygame.event.get() 
+   and is therefore similar to checking KEYDOWNS in the event loop.
+
+   .. note::
+      If you require getting the key presses in order. Use the event queue KEYDOWN events
+
+   ::
+      if pygame.key.get_just_pressed()[pygame.K_b]:
+         print("B key pressed")
```

Comment:
nitpick, could say "B key just pressed" or "B key pressed in the last game loop"